alert_group_nl_log2slack.py

```diff
@@ -181,18 +185,20 @@ def dump_cookies(session, where):
 def login_and_fetch(klant_nummer, klant_gecrypt):
     # Session keeps cookies around.
     with requests.Session() as session:
-        ret = session.get(ALERTMOBILE_URL)
+        ret = session.get(ALERTMOBILE_URL, timeout=10)
         dump_cookies(session, 'first get')
         assert ret.status_code == 200, (ret, ret.text)
 
         ret = session.post(ALERTMOBILE_URL, data={
                 'klantnr': klant_nummer, 'klantcode': '',
-                'gecrypt': klant_gecrypt})
+                'gecrypt': klant_gecrypt}, timeout=10)
```

Comment:
This is just slightly off-putting indentation now. You'll realise why. (Putting the timeout before data would've solved it. Or alternatively more lines.)

But leave it for the next round.

Reply:
In my head I had considered both options, but at least for the last one I thought I might get flack for using an extra line when it could fit on a single one.

I'll keep it in mind for the future though!